Implement the Cert Rotation Production deployment content changes (#292)
18F/identity-dev-docs: _pages/production.md

-   ATO approved environment

> -   [Approved agency logo](https://developers.login.gov/design-guidelines/#agency-logo-guidelines) for your app

Reviewer: Since this is the same repo, we can use baseurl here instead of hardcoding the domain, that way the links work in preview builds
```suggestion
-   [Approved agency logo]({{site.baseurl}}/design-guidelines/#agency-logo-guidelines) for your app
```

**When you have the components required, follow these steps to create your production configuration app:**

> 1.  [Create a new appon the](https://dashboard.int.identitysandbox.gov/) Login.gov Partner Dashboard. Select “Apps” from the top right menu, then select the “Create a new test app” button. 

Reviewer: typo, trailing space

```suggestion
1.  [Create a new app on the](https://dashboard.int.identitysandbox.gov/) Login.gov Partner Dashboard. Select “Apps” from the top right menu, then select the “Create a new test app” button.
```

8.  Enter the issuer, a unique string to identify the app in the **Identity Provider (IdP).** 

> 9.  Upload your logo and public certificate file using the "Choose a file" buttons.Note: the public certificate file may not be required if you are using the OIDC PKCE flow.

Reviewer: space between sentences
```suggestion
9.  Upload your logo and public certificate file using the "Choose a file" buttons. Note: the public certificate file may not be required if you are using the OIDC PKCE flow.
```
